system/Config/Services.php

```diff
@@ -179,7 +179,10 @@ public static function curlrequest(array $options = [], $response = null, \Confi
 		}
 
 		return new \CodeIgniter\HTTP\CURLRequest(
-				$config, new \CodeIgniter\HTTP\URI(), $response, $options
+				$config,
+				new \CodeIgniter\HTTP\URI(isset($options['base_uri']) ? $options['base_uri'] : null),
```

Comment:
Please use the shorter `?:` syntax here: `$options['base_uri'] : null` should work.

Reply:
But `isset($options['base_uri']) ?: null;` returns `true` or `null`.